console-frontend/src/pages/devices/index.rs

```diff
@@ -273,7 +272,7 @@ impl Index {
     }
 
     fn load_apps(&self, ctx: &Context<Self>) -> Result<RequestHandle, anyhow::Error> {
```

Comment:
If reqwest can be built to WASM target, we can use drogue-client here as well

Reply:
True. But I would like to do that as a follow up task. This PR is already rather big, bigger than anticipated.